addons/pvr.mediaportal.tvserver/src/Socket.cpp

```diff
@@ -138,7 +138,7 @@ bool Socket::bind ( const unsigned short port )
   _sockaddr.sin_addr.s_addr = INADDR_ANY;  //listen to all
   _sockaddr.sin_port = htons( port );
 
-  int bind_return = ::bind(_sd, (sockaddr*)(&_sockaddr), sizeof(_sockaddr));
+  int bind_return = MPTV::bind(_sd, reinterpret_cast<sockaddr*>(&_sockaddr), sizeof(_sockaddr));
```

Comment:
ha guys ... YES the fuckin bind was member of the MPT namespace - because of this include INSIDE the namespace MPT:

https://github.com/Memphiz/xbmc-pvr-addons/blob/c%2B%2B11-osx/addons/pvr.mediaportal.tvserver/src/Socket.h#L53

...